hw/ip/spi_host/data/spi_host.hjson

```diff
@@ -0,0 +1,592 @@
+      hwaccess: "hro",
+      fields: [
+        { bits: "31",
+          name: "SPIEN",
+          desc: '''Enables the SPI host.  On reset, this field is 0, meaning
+                   that no transactions can proceed.'''
+          resval: "0x0"
+        },
+        { bits: "30",
+          name: "RST_FSM",
+          desc: '''Holds the control FSM in reset when set to 1. (Must be
+                   explicitly cleared again before commands can be issued.)'''
+          resval: "0x0",
+          hwaccess: "hrw"
+        },
+        { bits: "29",
+          name: "RST_TXFIFO",
+          desc: '''Holds the TX FIFO in reset when set to 1. (Must be explicitly
+                   cleared again before using the TX FIFO.)'''
+          resval: "0x0",
+        },
+        { bits: "28",
+          name: "RST_RXFIFO",
+          desc: '''Holds the RX FIFO in reset when set to 1.  (Must be explicitly
+                   cleared again before using the RX FIFO.)'''
+          resval: "0x0"
+        },
+        { bits: "27",
+          name: "PASSTHRU",
+          desc: '''Enables Pass-through mode, wherein the SPI_HOST IP surrenders
+                   control of the SPI bus to another block. In this mode the
+                   `sck`, `cs_n` and `sd` signals are no longer driven by
+                   the SPI_HOST IP, rather they are multiplexed out to match the
+                   on-chip inputs `pt_sck_i`, `pt_cs_n_i`, `pt_sden_i`, `pt_sdo_i`,
+                   and `pt_sdi_o`.  (Since the `sd` bus is bidirectional, there are
+                   separate input, output and tri-state controls for this bus.)''',
+          resval: "0x0"
+        },
+        { bits: "24:16",
+          name: "TX_WATERMARK"
+          desc: '''If !!EVENT_ENABLE.TXWM is set, the IP will send
+                   an interrupt when the depth of the RX FIFO drops below
+                   TX_WATERMARK words (32b each).'''
+          resval: "0"
+        },
+        { bits: "15:9",
+          name: "RX_WATERMARK"
+          desc: '''If !!EVENT_ENABLE.RXWM is set, the IP will send
+                   an interrupt when the depth of the RX FIFO reaches
+                   RX_WATERMARK words (32b each).'''
+          resval: "127"
+        },
+        { bits: "8",
+          name: "MANCS_EN",
+          desc: '''Enable Manual Chip Select. Typically each command
+                   manages the CS lines automatically, issuing a command
+                   to exactly one device, and lowering and raising the
+                   appropriate CS line after transmitting or receiving the
+                   necessary bytes.  Asserting this bit overrides this
+                   automatic selection, instead setting the CS lines to
+                   reflect the contents of the !!CONTROL.MANUAL_CS field.'''
+          resval: "0x0"
+        },
+        { bits: "7:0",
+          name: "MANUAL_CS"
+          desc: '''When MANCS_EN = 1, the cs_n lines are tied directly to the
```

Comment:
Nit: Not that it's hard to find, but maybe link to `!!CONTROL.MANCS_EN` ?

Reply:
good idea. Done.

Reply:
Done!